python.ps1

```diff
@@ -1,3 +1,2 @@
 Write-Host "Installing python"
 & choco.exe install -y python
-refreshenv
```

Comment:
I can't recall if there's another refreshenv somewhere in the scripts. What you say should be true, because I recall Packer documentation claims that each script is launched in a pristine shell, so PATH etc. should get updated. However, in practice that did not seem to be the case. But if removing this does not break anything, I'm ok with it.

Reply:
It definitely doesn't help here at all. The default refreshenv only works for cmd.exe anyway and you use only powershell. You first need to import the choco powershell stuff to replace refreshenv with a powershell version. So I'm pretty sure this is a noop here. And yeah, didn't break anything for me.